### ISSUE TITLE
Windows, display flashes white, can't create black display

### ISSUE THREAD

**Issue by SimonN** [liballeg/allegro5#635]
I would like to generate a graphical window that's black on creation. I have success on X with that.

On Windows, the graphical window from `al_create_display` is white, not black. I know that A5's background buffer is fully black, so my best attempt is to flip the display immediately:

    display = al_create_display(/*...*/);
    al_flip_display();

Despite the flip, the newly-created window flashes white for a short time before becoming black. This short time is so short that sometimes, only half the screen flashes white. Nonetheless, the white flash is irritating.

* * *
Ideas from IRC earlier today:

workkeef: though if you really wanted I suspect you could probably hack in the internals a bit to clear it up
elias: https://github.com/liballeg/allegro5/blob/master/src/win/wwindow.c#L942
elias: maybe if you set the background to black instead of NULL?
SimonNa: thanks for the idea
SimonNa: I lack a Windows machine to build or test, normal Windows workflow: I cross-compile against pre-built DLLs and test at my friend's

**Comment by goobliata** [comment]
Here's code I used to do this in an SDL project of mine, run this right after the HWND is created:

```
PAINTSTRUCT ps;
HDC hdc;

hdc = BeginPaint(GetActiveWindow(), &ps);

SelectObject(hdc, GetStockObject(DC_BRUSH));
SetDCBrushColor(hdc, RGB(0, 0, 0));

Rectangle(hdc, 0, 0, win_w, win_h);
```

**Comment by goobliata** [comment]
This is fixed in 49a9151210f3eefda35ad6882a4fe2e1ecd67578.
